test/common/http/http2/codec_impl_test.cc

```diff
@@ -758,7 +769,9 @@ class Http2CodecImplStreamLimitTest : public Http2CodecImplTest {};
 // TODO(PiotrSikora): add tests that exercise both scenarios: before and after receiving
 // the HTTP/2 SETTINGS frame.
 TEST_P(Http2CodecImplStreamLimitTest, MaxClientStreams) {
+  ENVOY_LOG_MISC(error, "Client http2 settings: ");
```

Comment:
Is this needed at error level? Can you just run the test with debug logging to see (and print at that level if not already)?

Reply:
Because the settings values are randomly selected, if a test fails, using error level logging means we know which settings combination triggers the error even if the test doesn't run with debug binary. For example, if the test fails in continuous opt build, we can easily reproduce the failure by setting the values in the log.